### PR TITLE
Avoid thermostat card interaction on scroll on touch devices

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Makes the target dots the only draggable elements on touch devices for thermostat card and humidifier to avoid interaction when scrolling. More info is not changed.

Tested on : 
- Safari desktop
- Safari mobile (iOS)
- Firefox desktop
- Chrome desktop
- Chrome mobile (Android)

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-control-circular-slider.ts [hunk 1]
```diff
@@ -171,6 +173,18 @@ export class HaControlCircularSlider extends LitElement {
...
       }
     }
+
+    if (
+      (changedProps.has("_localValue") &&
+        changedProps.get("_localValue") == null) ||
+      (changedProps.has("_localLow") &&
+        changedProps.get("_localLow") == null) ||
+      (changedProps.has("_localHigh") &&
+        changedProps.get("_localHigh") == null) ||
```
Comment (bramkragten) [inline src/components/ha-control-circular-slider.ts:183]: Why do you do these checks? Is it to catch the first update?

File: src/components/ha-control-circular-slider.ts [hunk 2]
```diff
@@ -448,12 +475,17 @@ export class HaControlCircularSlider extends LitElement {
...
-    if (this._mc) {
-      this._mc.destroy();
-      this._mc = undefined;
+    if (this._managers.length > 0) {
+      this._managers.forEach((manager) => manager.destroy());
+      this._managers = [];
     }
   }
 
+  updateListeners() {
```
Comment (bramkragten) [inline src/components/ha-control-circular-slider.ts:484]: Why is this a public function?

File: src/components/ha-control-circular-slider.ts [hunk 3]
```diff
@@ -448,12 +475,17 @@ export class HaControlCircularSlider extends LitElement {
   }
 
   destroyListeners() {
```
Comment (bramkragten) [inline src/components/ha-control-circular-slider.ts:477]: Same for this btw, why public?

File: src/components/ha-control-circular-slider.ts (hunk 2 above)
Comment (piitaya) [reply src/components/ha-control-circular-slider.ts:484]: It should not 😅

File: src/components/ha-control-circular-slider.ts (hunk 1 above)
Comment (piitaya) [reply src/components/ha-control-circular-slider.ts:183]: The target dot is removed from the dom if the values are undefined so we need to recreate hammerjs.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**